cou/cli.py

```diff
@@ -89,6 +89,28 @@ def get_log_level(quiet: bool = False, verbosity: int = 0) -> str:
     return VerbosityLevel(verbosity).name
 
 
+async def continue_upgrade() -> bool:
+    """Determine whether to continue with the upgrade based on user input.
+
+    :return: Boolean value indicate whether to continue with the upgrade.
+    :rtype: bool
+    """
+    prompt_input = (
+        await ainput(prompt_message("Would you like to start the upgrade?", default_choice="n"))
+    ).casefold()
```

Comment:
Now I realized, what is the purpose of defualt_choise if we do not use it?
```suggestion
    default_choice = "n"
    prompt_input = await ainput(prompt_message("Would you like to start the upgrade?", default_choice="n"))
    prompt_input = (prompt_input or default_choice).casefold()
```